vivarium/library/topology.py

```diff
@@ -149,7 +149,7 @@ def dict_to_paths(root, d):
         return [(root, d)]
 
 
-def inverse_topology(outer, update, topology, inverse=None):
+def inverse_topology(outer, update, topology, inverse=None, initial=False):
```

Comment:
It feels like the `multi_update` logic should be the exception for when we're inverting updates, and the default should be the behavior for generating initial states (i.e. no `multi_update`s). That's technically a breaking change though. What do you think @eagmon?

Reply:
Hmm, so you are suggesting `initial=True` as the default? Yeah, that could break some things. I think it is fine as-is.

Reply:
Maybe change the flag to 'muli_updates=True' so that it is the flag, but still keep it true to avoid API change?

Reply:
In 97911c5, I gave the flag a more intuitive name without changing the API.